app/dao/fact_billing_dao.py

```diff
@@ -70,11 +70,13 @@ def fetch_sms_billing_for_all_services(start_date, end_date):
     # ASSUMPTION: AnnualBilling has been populated for year.
     ft_billing_subquery = query_sms_usage_for_year_per_service(financial_year).subquery()
 
-    sms_billable_units = func.sum(func.coalesce(ft_billing_subquery.c.chargeable_units, 0))
+    sms_billable_units = func.sum(ft_billing_subquery.c.chargeable_units)
```

Comment:
the coalesces are now back in. they ended up getting removed in the commit cos i'd removed them when debugging the raw sql to make it easier for me to read.

i think we need the coalesce on `sms_cost` because we say [`if float(s.sms_cost) > 0`](https://github.com/alphagov/notifications-api/blob/master/app/platform_stats/rest.py#L88) but we probably don't need it on any of the other points

Reply:
Hmm...I'm not sure. Is there a test that can expose which coalesces we actually need?

Looking at the query, the coalesces might be necessary if a service doesn't have any rows in `ft_billing` i.e. "Missing: no data for a service (no row in results)". [We do have such a test for the organisation report](https://github.com/alphagov/notifications-api/blob/e604385e0cf4c2ab8c6451b7120ceb196cce21b5/tests/app/dao/test_fact_billing_dao.py#L942), but that code works in a very different way so I think it's a stretch to say it's covered already.

Reply:
if a service doesn't have any ft_billing then for the platform admin report they won't be returned at all - it only returns those services with non-zero data.